sentry-ruby/spec/contexts/with_request_mock.rb

```diff
@@ -12,7 +12,13 @@ def setsockopt(*args); end
     allow(TCPSocket).to receive(:open).and_return(FakeSocket.new)
   end
 
-  def stub_request(fake_response, &block)
+  around do |example|
+    WebMock.disable!
+    example.run
+    WebMock.enable!
+  end
+
+  def sentry_stub_request(fake_response, &block)
     allow_any_instance_of(Net::HTTP).to receive(:connect)
```

Comment:
Do we still need these stubs with webmock?

Reply:
@st0012 this is plain RSpec mocking, not WebMock

Reply:
Ah we're not replacing the current stubbing completely. Sorry that I didn't go through the descriptions thoroughly:

> and keeps using our customized Net::HTTP stubbing context untouched and used the same way in specs where it is needed.